crates/astria-sequencer-utils/Cargo.toml

```diff
@@ -9,7 +9,7 @@ repository = "https://github.com/astriaorg/astria"
 homepage = "https://astria.org"
 
 [dependencies]
-base64 = { workspace = true }
+base64 = "0.22.1"
```

Comment:
I guess the base64 encoded blob data we feed into the blob parser command is not controlled by us?